config/routes.rb

```diff
@@ -1,4 +1,11 @@
 Rails.application.routes.draw do
+  devise_for :users, controllers: {
+    sessions: 'users/sessions',
+    registrations: 'users/registrations'
```

Comment:
@xiaopow You are adding [a lot of modules](https://github.com/WDI-HK-7/Coconut/pull/12/files#diff-4676c008b11a5480d73d4a6de01e45b9R4) to the `User` model, but only generating routes to `sessions` and `registrations`